python/spear/game_world_service.py

```diff
@@ -92,11 +111,282 @@ def get_stable_name_for_actor_component(self, component, include_actor_name):
     def get_stable_name_for_scene_component(self, component, include_actor_name):
         return self._rpc_client.call("game_world_service.get_stable_name_for_scene_component", component, include_actor_name)
 
+    #
+    # Get actor and component tags
+    #
+
     def get_actor_tags(self, actor):
         return self._rpc_client.call("game_world_service.get_actor_tags", actor)
 
     def get_component_tags(self, actor):
         return self._rpc_client.call("game_world_service.get_component_tags", actor)
 
+    #
+    # Get class
+    #
+
     def get_class(self, uobject):
         return self._rpc_client.call("game_world_service.get_class", uobject)
+
+    #
+    # Create components
+    #
+
+    def create_component_on_actor(self, class_name, owner, name):
+        return self._rpc_client.call("game_world_service.create_component_outside_owner_constructor", class_name, owner, name)
+
+    def create_scene_component_on_actor(self, class_name, owner, name):
+        return self._rpc_client.call("game_world_service.create_scene_component_outside_owner_constructor_from_actor", class_name, owner, name)
+
+    def create_scene_component_on_component(self, class_name, owner, parent, name):
+        return self._rpc_client.call("game_world_service.create_scene_component_outside_owner_constructor_from_object", class_name, owner, parent, name)
+
+    def create_scene_component_on_owning_scene_component(self, class_name, owner, name):
```

Comment:
- Need `destroy_actor(self, actor)` immediately after `spawn_actor`.
- Need `destroy_component(self, component, promote_children=False)` immediately after our functions for creating components.
- There is a `K2_DestroyComponent()` `UFUNCTION`, but this function always sets `promote_children` to false, which feels too limiting.
- There is also a `K2_DestroyActor()` `UFUNCTION`, so we could insist that users use this function. But it feels too confusing/asymmetric/inconsistent if we have different approaches for creating and destroying. And it feels too inconsistent if we have a different approaches for actors (i.e., where we could use the `UFUNCTION`) and components (i.e., where we need to use our own function to expose the `promote_children` flag).